source/logic.rst

```diff
@@ -0,0 +1,492 @@
+.. _logic:
+
+Logic
+=====
+
+In the last chapter, we dealt with equations, inequalities,
+and basic mathematical statements like
+":math:`x` divides :math:`y`."
+Complex mathematical statements are built up from
+simple ones like these
+using logical terms like "and," "or," "not," and
+"if ... then," "every," and "some."
+In this chapter, we show you how to work with statements
+that are built up in this way.
+
+
+.. _using_implication_and_the_universal_quantifier:
+
+Implication and the Universal Quantifier
+----------------------------------------
+
+Consider the statement after the ``#check``:
+
+.. code-block:: lean
+
+    import data.real.basic
+
+    -- BEGIN
+    #check ∀ x y ε : ℝ,
+      0 < ε → ε ≤ 1 → abs x < ε → abs y < ε → abs (x * y) < ε
+    -- END
+
+In words, we would say "for every ``x``, ``y``, and ``ε``,
+if ``0 < ε ≤ 1``, the absolute value of ``x`` is less than ``ε``,
+and the absolute value of ``y`` is less than ``ε``,
+then the absolute value of ``x * y`` is less than ``ε``."
+In Lean, the implication arrows associate to the right,
+making it easy to chain implications in this way.
```

Comment:
Mathematicians don't know what "associates to the right means". We need to have an example like: "P → Q → R" means 'P → (Q → R)". And even with parentheses this is still hard to parse for regular mathematicians.